tests/template/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -63,17 +63,19 @@ g
 }\`;
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 \`
-(?:\${escapeChar}[\\\\S\\\\s]|(?:(?!\${// Using \`XRegExp.union\` safely rewrites backreferences in \`left\` and \`right\`.
-// Intentionally not passing \`basicFlags\` to \`XRegExp.union\` since any syntax
-// transformation resulting from those flags was already applied to \`left\` and
-// \`right\` when they were passed through the XRegExp constructor above.
-XRegExp.union([left, right], "", { conjunction: "or" })
-  .source})[^\${escapeChar}])+)+
+(?:\${escapeChar}[\\\\S\\\\s]|(?:(?!\${
+  // Using \`XRegExp.union\` safely rewrites backreferences in \`left\` and \`right\`.
+  // Intentionally not passing \`basicFlags\` to \`XRegExp.union\` since any syntax
+  // transformation resulting from those flags was already applied to \`left\` and
+  // \`right\` when they were passed through the XRegExp constructor above.
+  XRegExp.union([left, right], "", { conjunction: "or" }).source
+})[^\${escapeChar}])+)+
 \`;
 
-\`a\${/* b */ c /* d */}e\${// f
-g
-// h
+\`a\${/* b */ c /* d */}e\${
+  // f
```

Comment:
I could see someone opening an issue because this comment was moved, but in the XRegExp example, moving it is way better. Not sure how to best handle that; maybe we should treat multiline comments differently from single-line comments? Might not actually be a big deal though.

Reply:
I agree. I think it's a good tradeoff for keeping the logic simple but it's possible we could find a good heuristic for these cases

Reply:
Let's merge this as-is for now, and if more issues are opened, those might help us come up with a heuristic.

Reply:
I'm not too worried about comments here, they are extremely rare and this output looks decent.